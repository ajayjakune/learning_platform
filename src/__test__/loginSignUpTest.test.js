import React from 'react';
import ReactDom from 'react-dom';
import { render, cleanup } from '@testing-library/react';
import LoginSignUp from '../components/Auth/loginSignUp';

afterEach(cleanup)

it("renders without crashing", () => {
    const { asFragment } = render(<LoginSignUp />)

    expect(asFragment(<LoginSignUp />)).toMatchSnapshot()
})
