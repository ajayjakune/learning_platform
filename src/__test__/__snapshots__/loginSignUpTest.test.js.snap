// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="row"
  >
    <div
      class="col-md-6 mx-auto p-0"
    >
      <div
        class="card"
      >
        <div
          class="login-box"
        >
          <div
            class="login-snip"
          >
            <input
              checked=""
              class="sign-in"
              id="tab-1"
              name="tab"
              type="radio"
            />
            <label
              class="tab"
              for="tab-1"
            >
              Login
            </label>
            <input
              class="sign-up"
              id="tab-2"
              name="tab"
              type="radio"
            />
            <label
              class="tab"
              for="tab-2"
            >
              Sign Up
            </label>
            <div
              class="login-space"
            >
              <div
                class="login"
              >
                <form>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="user"
                    >
                      Username
                    </label>
                    <input
                      class="input"
                      name="email"
                      placeholder="Enter email address"
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="pass"
                    >
                      Password
                    </label>
                    <input
                      class="input"
                      data-type="password"
                      name="password"
                      placeholder="Enter password"
                      type="password"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <input
                      class="button"
                      type="submit"
                      value="Sign In"
                    />
                  </div>
                  <div
                    class="hr"
                  />
                </form>
              </div>
              <div
                class="sign-up-form"
              >
                <form>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="user"
                    >
                      First Name
                    </label>
                    <input
                      class="input"
                      name="firstName"
                      placeholder="Enter first name"
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="user"
                    >
                      Last Name
                    </label>
                    <input
                      class="input"
                      name="lastName"
                      placeholder="Enter last name"
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="user"
                    >
                      Email Address
                    </label>
                    <input
                      class="input"
                      name="email"
                      placeholder="Enter email address"
                      type="text"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <label
                      class="label"
                      for="pass"
                    >
                      Password
                    </label>
                    <input
                      class="input"
                      data-type="password"
                      name="password"
                      placeholder="Create your password"
                      type="password"
                      value=""
                    />
                  </div>
                  <div
                    class="group"
                  >
                    <input
                      class="button"
                      type="submit"
                      value="Sign Up"
                    />
                  </div>
                  <div
                    class="hr"
                  />
                  <div
                    class="foot"
                  >
                     
                    <label
                      for="tab-1"
                    >
                      Already Member?
                    </label>
                     
                  </div>
                </form>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
